nextflow_schema.json

```diff
@@ -689,128 +929,131 @@
+                    "fa_icon": "fas fa-angle-double-down",
+                    "default": true
                 }
             },
-            "fa_icon": "fas fa-tools"
+            "fa_icon": "fas fa-bacteria"
         },
         "arg_abricate": {
             "title": "ARG: ABRicate",
             "type": "object",
-            "description": "Antimicrobial resistance gene detection, based on alignment to CBI, CARD, ARG-ANNOT, Resfinder, MEGARES, EcOH, PlasmidFinder, Ecoli_VF and VFDB.",
+            "description": "Antimicrobial resistance gene detection based on alignment to CBI, CARD, ARG-ANNOT, ResFinder, MEGARES, EcOH, PlasmidFinder, Ecoli_VF and VFDB. More info: https://github.com/tseemann/abricate",
             "default": "",
-            "fa_icon": "fas fa-tools",
+            "fa_icon": "fas fa-bacteria",
             "properties": {
                 "arg_skip_abricate": {
                     "type": "boolean",
                     "fa_icon": "fas fa-ban",
-                    "description": "Skip ABRicate during the ARG-screening."
+                    "description": "Skip ABRicate during the ARG screening."
                 },
-                "arg_abricate_db": {
+                "arg_abricate_db_id": {
                     "type": "string",
                     "default": "ncbi",
-                    "fa_icon": "fas fa-layer-group",
-                    "description": "Specify which of the provided public databases to use by ABRicate.",
-                    "enum": [
-                        "argannot",
-                        "card",
-                        "ecoh",
-                        "ecoli_vf",
-                        "megares",
-                        "ncbi",
-                        "plasmidfinder",
-                        "resfinder",
-                        "vfdb",
-                        "bacmet2",
-                        "victors"
-                    ],
-                    "help_text": "Specifies which database to use from dedicated list of databases available by ABRicate. \n\nFor more information check ABRicate [documentation](https://github.com/tseemann/abricate).\n\n> Modifies tool parameter(s):\n> - ABRicate: `--db`"
+                    "fa_icon": "fas fa-database",
+                    "description": "Specify the name of the ABRicate database to use. Names of non-default databases can be supplied if `--arg_abricate_localdbdir` provided.",
+                    "help_text": "Specifies which database to use from dedicated list of databases available by ABRicate.\n\nDefault supported are one of: `argannot`, `card`, `ecoh`, `ecoli_vf`, `megares`, `ncbi`, `plasmidfinder`, `resfinder`, `vfdb`. Other options can be supplied if you have installed a custom one within the directory you have supplied to `--arg_abricate_db`.\n\nFor more information check the ABRicate [documentation](https://github.com/tseemann/abricate).\n\n> Modifies tool parameter(s):\n> - ABRicate: `--db`"
```

Comment:
maybe add an enum field?

Reply:
The DB name can be one of those (argannot, card etc.) but also custom ones, which the user has to type themselves, so I think enum list cannot be used here.